### PR TITLE
IOF-343 log: Update to gurt log v2

### PR DESCRIPTION
**Pull request by jolivier23**
Signed-off-by: Jeff Olivier <jeffrey.v.olivier@intel.com>

### CODE REVIEW

**Comment by jolivier23** [comment]
any idea why this won't run?

**Comment by ashleypittman** [comment]
> any idea why this won't run?

I'd merged the branch you based it on so I rebased this to master.  It likely won't run as it depends on the new v2 log stuff in CaRT and IOF is using an older version of CaRT still due to the stability issues.

**Comment by jolivier23** [comment]
I meant why doesn’t the build run.  I don’t expect it to pass testing.

**Comment by ashleypittman** [comment]
> I meant why doesn’t the build run. I don’t expect it to pass testing.

I logged in and had a look, there was a problem with cleaning the workspace on previous builds.  For a while master was broken but we fixed it shortly afterwards, it so happened that when I rebased this was within the window where it didn't work.

File: test/common_methods.py [hunk 1]
```diff
@@ -753,7 +753,7 @@ def test_many_files(self):
 
         test_dir = os.path.join(self.import_dir, 'many')
         files = []
-        for x in range(0, 100):
+        for x in range(0, 15):
```
Comment (ashleypittman) [inline test/common_methods.py]: This isn't needed now a the bug has been fixed in CaRT.  I'll update the CaRT ref and then we can just process the log changes in this PR.

File: setup_local.sh [hunk 2]
```diff
@@ -34,13 +34,12 @@
 # ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-VARS_FILE=./.build_vars-`uname -s`.sh
+VARS_FILE=./.build_vars.sh
```
Comment (jolivier23) [inline setup_local.sh]: Is this file any different from ./scons_local/utils/setup_local.sh ?

Comment (ashleypittman) [reply setup_local.sh]: Yes, unfortunately it is.  I could back out this part of the PR, but simply removing this file and using the alternative isn't that easy.

Comment (jolivier23) [reply setup_local.sh]: ok, no problem.  Not worth it.

**Pull request merged**